src/pages/Cart/cart.css

```diff
@@ -0,0 +1,129 @@
+.cart-cont {
+  display: flex;
+  gap: 2rem;
+  padding: 2rem 1rem;
+  justify-content: center;
+}
+.cart-left-cont {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+.cart-right-cont {
+  width: 340px;
+}
+.cart-item {
+  max-width: 36rem;
+  min-height: 13rem;
+  max-height: 13rem;
+  margin-bottom: 1rem;
+  display: flex;
+  height: auto;
+  overflow: hidden;
+  border-radius: 0.5rem;
+  border: 1px solid var(--COLOR-GRAY-100);
+}
+
+.cart-item .cart-item-img {
+  max-width: 40%;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.cart-item-img img {
+  object-fit: cover;
+  width: 80%;
+}
+.cart-item-details {
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.cart-item-details .cart-item-title {
+  text-transform: uppercase;
+  font-size: 0.8rem;
+  font-weight: 800;
+}
+.cart-item-details .cart-item-desc {
+  padding-bottom: 0.5rem;
+}
+.cart-item-price .price-new {
+  padding-right: 0.5rem;
+  font-weight: 400;
+}
+.cart-item-price .price-old {
+  font-weight: 300;
+  text-decoration: line-through;
+  padding-right: 5px;
+  opacity: 0.7;
+  color: var(--COLOR-GRAY-300);
+}
+.cart-item-price .discount {
+  color: var(--COLOR-PRIMARY);
+}
+.cart-item-footer {
+  display: flex;
+  align-items: center;
+  column-gap: 0.5rem;
+}
+.cart-item-footer .btn-qty {
+  font-size: 1.4rem;
+  font-weight: 300;
+  min-width: 0;
+  background-color: #95aaff1a;
```

Comment:
is this correct hex check once

Reply:
yes this is correct. its with opacity thats why it is long